<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowClass</class>
 <widget class="QMainWindow" name="MainWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>305</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Bach Nguyen | Pop Up Notification</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0" colspan="2">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="1">
       <widget class="QDoubleSpinBox" name="roundnessDsbx"/>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="lineLengthLbl">
        <property name="text">
         <string>Line length:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="opacityLbl">
        <property name="text">
         <string>Opacity:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QDoubleSpinBox" name="opacityDsbx"/>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="lineWidthLbl">
        <property name="text">
         <string>Line width:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QSpinBox" name="lineLengthSbx"/>
      </item>
      <item row="1" column="3">
       <widget class="QSpinBox" name="lineWidthSbx"/>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="roundnessLbl">
        <property name="text">
         <string>Roundness:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLabel" name="innerRadiusLbl">
        <property name="text">
         <string>Inner Radius:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QSpinBox" name="innerRadiusSbx"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="linesLbl">
        <property name="text">
         <string>Lines:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="fadePercLbl">
        <property name="text">
         <string>Fade Perc:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QDoubleSpinBox" name="fadePercDsbx"/>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="timeLbl_2">
        <property name="text">
         <string>X:</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QSpinBox" name="xSbx">
        <property name="maximum">
         <number>3000</number>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="colorLbl">
        <property name="text">
         <string>Color:</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="timeLbl">
        <property name="text">
         <string>Times (s):</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QSpinBox" name="linesSbx"/>
      </item>
      <item row="5" column="3">
       <widget class="QSpinBox" name="stepSbx">
        <property name="minimum">
         <number>5</number>
        </property>
        <property name="maximum">
         <number>900</number>
        </property>
        <property name="singleStep">
         <number>5</number>
        </property>
        <property name="value">
         <number>60</number>
        </property>
       </widget>
      </item>
      <item row="6" column="3">
       <widget class="QSpinBox" name="widthSbx">
        <property name="minimum">
         <number>10</number>
        </property>
        <property name="maximum">
         <number>1000</number>
        </property>
        <property name="value">
         <number>120</number>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QPushButton" name="colorBtn">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="5" column="2">
       <widget class="QLabel" name="stepLbl">
        <property name="text">
         <string>Step (s):</string>
        </property>
       </widget>
      </item>
      <item row="6" column="2">
       <widget class="QLabel" name="timeLbl_3">
        <property name="text">
         <string>Width:</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QSpinBox" name="timeSbx">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>60</number>
        </property>
        <property name="value">
         <number>2</number>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QLabel" name="revLbl">
        <property name="text">
         <string>Rev/S:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="QDoubleSpinBox" name="revDsbx"/>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="timeLbl_4">
        <property name="text">
         <string>Y:</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QSpinBox" name="ySbx">
        <property name="maximum">
         <number>3000</number>
        </property>
       </widget>
      </item>
      <item row="7" column="2">
       <widget class="QLabel" name="timeLbl_5">
        <property name="text">
         <string>Height:</string>
        </property>
       </widget>
      </item>
      <item row="7" column="3">
       <widget class="QSpinBox" name="heightSbx">
        <property name="minimum">
         <number>10</number>
        </property>
        <property name="maximum">
         <number>1000</number>
        </property>
        <property name="value">
         <number>50</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="showBtn">
      <property name="text">
       <string>Show</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="hideBtn">
      <property name="text">
       <string>Hide</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="startBtn">
      <property name="text">
       <string>Start</string>
      </property>
      <property name="autoDefault">
       <bool>true</bool>
      </property>
      <property name="default">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="exitBtn">
      <property name="text">
       <string>Exit</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>roundnessDsbx</tabstop>
  <tabstop>lineLengthSbx</tabstop>
  <tabstop>opacityDsbx</tabstop>
  <tabstop>lineWidthSbx</tabstop>
  <tabstop>fadePercDsbx</tabstop>
  <tabstop>innerRadiusSbx</tabstop>
  <tabstop>linesSbx</tabstop>
  <tabstop>revDsbx</tabstop>
  <tabstop>colorBtn</tabstop>
  <tabstop>timeSbx</tabstop>
  <tabstop>stepSbx</tabstop>
  <tabstop>xSbx</tabstop>
  <tabstop>ySbx</tabstop>
  <tabstop>widthSbx</tabstop>
  <tabstop>heightSbx</tabstop>
  <tabstop>showBtn</tabstop>
  <tabstop>hideBtn</tabstop>
  <tabstop>startBtn</tabstop>
  <tabstop>exitBtn</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
